Source/Urho3D/Engine/Console.cpp

```diff
@@ -376,14 +376,10 @@ void Console::HandleTextFinished(StringHash eventType, VariantMap& eventData)
         // Send the command as an event for script subsystem
         using namespace ConsoleCommand;
 
-#if URHO3D_CXX11
-        SendEvent(E_CONSOLECOMMAND, P_COMMAND, line, P_ID, static_cast<Text*>(interpreters_->GetSelectedItem())->GetText());
-#else
         VariantMap& newEventData = GetEventDataMap();

```

Comment:
I... I don't understand the issue, sorry. I removed block under #if guard because we don't use variadic SendEvent in any other place than this.